package.json

```diff
@@ -4,40 +4,41 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "watch": "nodemon --watch ./ --watch views/ --watch controllers/ --watch build/ --ext js,hbs ./bin/www",
+    "watch": "npx nodemon --watch ./ --watch views/ --watch controllers/ --watch build/ --ext js,hbs ./bin/www",
     "debug": "nodemon --inspect=0.0.0.0:9311 --watch controllers/ --watch helpers/ --watch views/ --ext js,hbs ./bin/www",
```

Comment:
If nodemon is missing locally for the watch script it's also missing for the debug script. Npx should be added there as well.

The alternative would be to add nodemon to devDependencies. Is this an option here?

Reply:
readme says: `npm install -g nodemon gulp`

Reply:
As you updated a lot more than jsonwebtoken anyway, couldn't you add nodemon and gulp as dependencies in package.json and remove this instruction from the readme? Or if you don't wanna do it in this ticket, shall we make a new small ticket for this change? I believe it's a rather bad practice to install them globally.